contrib/hyperkube-image/Makefile

```diff
@@ -0,0 +1,20 @@
+TAG = v1.1.2
+PREFIX = quay.io/coreos
+
+.PHONY: all container push clean
+
+all: container
+
+hyperkube:
+	#TODO(aaron): build from source?
```

Comment:
Good point. And we currently do have patches we are carrying for the Kubelet in the OS build.